pkg/core/localize/locales/en/cmd/rule.en.toml

```diff
@@ -3,47 +3,48 @@ one = 'Manage artifact rules in a Service Registry instance'
 
 [registry.rule.cmd.description.long]
 one = '''
-Set of commands for managing validity and compatibility rules for an entire Service Registry instance or individual for each artifact.
-Service Registry rules govern how artifact content can evolve over time by validating for any artifact updates
+Configure the validity and compatibility rules that govern artifact content.
 
-Rules can be configured as global rules for entire Service Registry instances or specific artifacts.
+When you add or update an artifact, Service Registry applies rules to check the validity and compatibility of the artifact content. Artifact-specific rules apply to the specified artifact only. Global rules apply to all artifacts in a particular Service Registry instance. Configured artifact-specific rules override any configured global rules. Before a new artifact version can be uploaded to the registry, all configured global rules or artifact-specific rules must pass.
+
+For more information about supported Service Registry content and rules, see https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817.
 '''
```

Comment:
Suggest not repeating the information already provided in `rhoas_service-registry_rule.md` for minimalism.

Reply:
Fixed in the rule.en.toml file. (Note that the docs/commands/x.md files are modular docs that are generated from the source pkg/core/localize/locales/en/cmd/y.toml files. I should have mentioned that you didn't need to review the .md files, my apologies!)